src/Microsoft.IdentityModel.Abstractions/DataClassification.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.IdentityModel.Abstractions
+{
+    /// <summary>
+    /// Data classification categories defined by the asset classification standard.
+    /// </summary>
+    /// <remarks>
+    /// See http://aka.ms/AssetClassificationStandard for more details about the asset classification standard.
+    /// </remarks>
+    public enum DataClassification
+    {
+        /// <summary>
+        /// Data used to manage access to administrative roles or sensitive functions.
+        /// </summary>
+        AccessControlData = 0,
```

Comment:
From https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum:

```
✔️ DO provide a value of zero on simple enums.

Consider calling the value something like "None." If such a value is not appropriate for this particular enum, the most common default value for the enum should be assigned the underlying value of zero.
```

Reply:
I see 0 as highest PII as the right model here, makes it privacy strict by default. If someone doesn't think about it and default is NoPII then we may have a privacy breach by carelessness, this defaults to more obfuscated metrics by default which is far better IMO.